go/service/account.go

```diff
@@ -164,3 +164,19 @@ func (h *AccountHandler) SetLockdownMode(ctx context.Context, arg keybase1.SetLo
 	_, err = h.G().API.Post(apiArg)
 	return err
 }
+
+func (h *AccountHandler) TestPassphrase(ctx context.Context, sessionID int) (err error) {
+	mctx := libkb.NewMetaContext(ctx, h.G())
+	defer mctx.Trace("TestPassphrase", func() error { return err })()
+
+	username := h.G().GetEnv().GetUsername().String()
+
+	arg := libkb.DefaultPassphrasePromptArg(mctx, username)
+	secretUI := h.getSecretUI(sessionID, h.G())
+	res, err := secretUI.GetPassphrase(arg, nil)
```

Comment:
@patrickxb can you take a look? for simplicity it would be preferable to pass the passphrase over as argument to this RPC, but are we fine with that?

Reply:
Wouldn't we want to use pinentry?  Which secret ui will handle.